unittests/test.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<espa_metadata version="1.3"
-xmlns="http://espa.cr.usgs.gov/v1"
+<espa_metadata version="2.0"
```

Comment:
i agree with @raydittmeier . perhaps it could live in a single file, (version.py ?) and just have values pulled in from there.  its a common practice